lib/assets/data_table/main.css

```diff
@@ -0,0 +1,187 @@
+.app {
+  font-family: "Inter";
+
+  padding: 0px 12px 12px 12px;
+
+  --gray-50: #f8fafc;
+  --gray-100: #f0f5f9;
+  --gray-200: #e1e8f0;
+  --gray-300: #cad5e0;
+  --gray-400: #91a4b7;
+  --gray-500: #61758a;
+  --gray-700: #304254;
+  --gray-800: #1c2a3a;
```

Comment:
Since the page is sandboxed we need to replicate some Livebook-specific CSS, but it's not that much. In the future we could have a tiny Livebook CSS package for people to use.